src/battle/move/unknown_move/779C90.c

```diff
@@ -1,7 +1,34 @@
 #include "common.h"
 
+#include "battle/battle.h"
+
 #define NAMESPACE battle_move_unknown_move
 
 #include "world/common/UnkMoveFunc1.inc.c"
 
-INCLUDE_ASM(s32, "battle/move/unknown_move/779C90", func_802A10E4_779D74);
+extern s32 D_802A26D0_77B360;
+
+ApiStatus func_802A10E4_779D74(Evt* script, s32 isInitialCall) {
+    Actor *enemyTargetActor;
```

Comment:
Pointer on the left pleez (Actor*) and also please move the enemyTargetActor initialization into this line as well